uimaj-core/src/main/java/org/apache/uima/resource/impl/PearSpecifier_impl.java

```diff
@@ -48,16 +47,16 @@ public PearSpecifier_impl() {
   /**
    * @return Returns the Parameters.
    */
-  public Parameter[] getParameters() {
+  public NameValuePair[] getPearParameters() {
```

Comment:
The old setters could continue to exist and the new setters could in fact user the same name as the old ones, just different argument types.

It would be great to allow for syntactic sugar by using a variadic argument instead of an array. This could even be changed for the old setters without introducing a breaking change (I think).